### PR TITLE
DO - 138 picture viewer issues

### PR DESCRIPTION
**Pull request by Palakagrawal28**

### CODE REVIEW

File: src/components/common/PictureViewer/_PictureViewer.scss [hunk 1]
```diff
@@ -1,21 +1,8 @@
...
-}
-.member-profile-picture-viewer__container {
-    max-height: calc(100vh - 80px);
-    width: 100%;
-    height: 400px;
-    min-width: 200px;
-    min-height: 200px;
-}
 .image-gallery-content .image-gallery-slide .image-gallery-image{
   width: 500px;
```
Comment (amitjain24524) [inline src/components/common/PictureViewer/_PictureViewer.scss]: Image gets crop If we use fix width and height.
Use 100% or auto for image.

File: src/components/common/PictureViewer/_PictureViewer.scss [hunk 2]
```diff
@@ -45,29 +28,11 @@
...
-  }
-  .image-gallery-thumbnails {
-    overflow: hidden;
-    padding: 25px 0;
-  }
-}
-@media (min-width: 1px) and (max-width: 767px) {
+@media (max-width: 1024px) {
   .image-gallery-content .image-gallery-slide .image-gallery-image{
     width: 885px;
```
Comment (amitjain24524) [inline src/components/common/PictureViewer/_PictureViewer.scss:41]: Image gets crop If we use fix width and height.
Use 100% or auto for image.

File: src/components/common/PictureViewer/_PictureViewer.scss (hunk 1 above)
Comment (Palakagrawal28) [reply src/components/common/PictureViewer/_PictureViewer.scss]: Some images looks fine with 100% whereas some images get cropped even on 100% / auto.

File: src/components/common/PictureViewer/_PictureViewer.scss (hunk 2 above)
Comment (Palakagrawal28) [reply src/components/common/PictureViewer/_PictureViewer.scss:41]: Some images looks fine with 100% whereas some images get cropped even on 100% / auto.

**Pull request closed without merging**